### PR TITLE
Adding support for pass phrase.

### PR DESCRIPTION
**Pull request by pavkatar**
Adding support for sending a passphrase when sending a local certificate.

### CODE REVIEW

File: src/AfriCC/EPP/Client.php [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Client
     protected $serviceExtensions;
     protected $ssl;
     protected $local_cert;
+    protected $pass_phrase;
```
Comment (johnny-bit) [inline src/AfriCC/EPP/Client.php]: Since ssl stream context option is called `passphrase` adding underscore here, is rather non-intuitive.

**Comment by johnny-bit** [comment]
Hi!

This change is rather straightforward and helpful, however `pass_phrase` instead of `passphrase` can get confusing :wink:

**Comment by lifeofguenter** [comment]
Awesome work @pavkatar - thanks for this! I agree though with @johnny-bit , to keep things consistent, can you please remove the underscore? I will merge right away after that small cosmetic change.

File: src/AfriCC/EPP/Client.php [hunk 2]
```diff
@@ -76,6 +77,9 @@ public function __construct(array $config)
             if (!is_readable($this->local_cert)) {
                 throw new Exception(sprintf('unable to read local_cert: %s', $this->local_cert));
             }
+            if (!empty($config['pass_phrase'])) {
```
Comment (lifeofguenter) [inline src/AfriCC/EPP/Client.php]: please add a newline above

File: src/AfriCC/EPP/Client.php [hunk 3]
```diff
@@ -116,6 +120,9 @@ public function connect()
 
             if ($this->local_cert !== null) {
                 stream_context_set_option($context, 'ssl', 'local_cert', $this->local_cert);
+                if($this->pass_phrase) {
```
Comment (lifeofguenter) [inline src/AfriCC/EPP/Client.php]: `if (`

**Pull request merged**